presto-docs/src/main/sphinx/release/release-0.281.rst

```diff
@@ -0,0 +1,58 @@
+=============
+Release 0.281
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix timeout for optimization rules.
```

Comment:
Can this be elaborated upon?  Is it a bug fix, or does it fix the timeout logic?